eo-parser/src/main/java/org/eolang/parser/XeEoListener.java

```diff
@@ -1298,12 +1298,11 @@ private Objects startAbstract(final ParserRuleContext ctx) {
     private void putComment(final List<EoParser.CommentContext> comment, final Token stop) {
         if (!comment.isEmpty()) {
             this.dirs.push().xpath("/program").addIf("comments").add("comment").set(
-                String.join(
-                    "",
-                    comment.stream().map(
-                        context -> context.COMMENTARY().getText().substring(1).trim()
-                    ).collect(Collectors.joining(""))
-                )
+                comment.stream().map(
+                    context -> context.COMMENTARY().getText().substring(1).trim()
+                ).map(
+                    text -> String.format("%s\\n", text)
+                ).collect(Collectors.joining(""))
```

Comment:
@h1alexbel can't we simplify this with just `collect(Collectors.joining("\n"))` and don't use `.map`?